api/tests/test_content_api.py

```diff
@@ -68,12 +67,12 @@ def tearDown(self):
     def test_set_up(self):
         """Assert that that the models were created in setUp()"""
         try:
-            user = User.objects.get(username = USER_A["username"])
-            user = User.objects.get(id = self.user_a.id)
+            user = User.objects.get(username=USER_A["username"])
+            User.objects.get(id=self.user_a.id)
         except:
             self.assertFalse(True, "Error retrieving %s from database" %USER_A["username"])
         try:
-            post = Post.objects.get(guid=self.post.guid)
+            Post.objects.get(guid=self.post.guid)
```

Comment:
Because it's not being used anywhere. We're really just testing if it throw an exception.